gui/widgets/matRad_GammaWidget.m

```diff
@@ -0,0 +1,567 @@
+classdef matRad_GammaWidget < matRad_Widget
+% matRad_GammaWidget : GUI widget for gamma index based comparisons of  
+% dose cubes stored within resultGUI struct.
+%
+% References
+%   -
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Copyright 2020 the matRad development team. 
+% 
+% This file is part of the matRad project. It is subject to the license 
+% terms in the LICENSE file found in the top-level directory of this 
+% distribution and at https://github.com/e0404/matRad/LICENSES.txt. No part 
+% of the matRad project, including this file, may be copied, modified, 
+% propagated, or distributed except according to the terms contained in the 
+% LICENSE file.
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%    
+
+
+    properties
+        SelectedDisplayOption1 = 'physicalDose' ;
+        SelectedDisplayOption2 = 'physicalDose' ;
+        SelectedDisplayAllOptions = strings;
+        criteria = [3 3];
+        n = 0;
+        localglobal = 'global';
+        resolution;
+        lockUpdate = false;
+        maxSlice;
+        slice;
+        gammaCube;
+        gammaPassRateCell;
+        updated = false;
+        
+    end 
+    properties (Constant)
+      normalization = {'local','global'}  ;
+    end 
+
+    events
+
+    end 
+
+    methods
+        function this  = matRad_GammaWidget(handleParent)
+%MANAGE ARGUMENTS 
```

Comment:
what is this comment for?

Reply:
I think im gonna get rid of Gamma widget from this PR. and submit it as a separate PR as its quite a mess